bundles/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/handler/ModbusDataThingHandler.java

```diff
@@ -441,7 +440,12 @@ public synchronized void initialize() {
                 comms = localPollerHandler.getCommunicationInterface();
                 pollStart = localReadRequest.getReference();
                 childOfEndpoint = false;
+            } else {
```

Comment:
Is this branch ever executed? Is it possible configure illegal bridge?

Reply:
I'm not sure, whether it is, but I feel that this way the code is easier to read. Before, the logic was somewhat more implicit and the reader had to read more and more carefully to understand that exactly these two options exist and what they are about. By having a separate branch with an explicit handling I feel it's a little bit easier to follow.